modular_skyrat/modules/cargo/code/goodies.dm

```diff
@@ -57,6 +57,12 @@
 	contains = list(/obj/item/gun_maintenance_supplies,
 					/obj/item/gun_maintenance_supplies)
 
+/datum/supply_pack/goody/sabre
+	name = "Sabre"
+	desc = "A sabre that comes in a ornate black sheath."
```

Comment:
Would be nice if it was called something else than just "Sabre", it's like saying "Handgun", I'm sure there can be something funnier to be done with that.